core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/controlcommands/ControlCommandConvertUtils.scala

```diff
@@ -49,7 +49,7 @@ object ControlCommandConvertUtils {
         ModifyOperatorLogicV2(code, isSource)
       case EvaluateExpression(expression) =>
         EvaluateExpressionV2(expression)
-      case QuerySelfWorkloadMetrics() =>
+      case QueryWorkloadMetrics() =>
         QuerySelfWorkloadMetricsV2()
```

Comment:
please keep V1 and V2 name synchronized. we can change it later.

Reply:
done the change